src/extension/importer/core/src/main/java/org/geoserver/importer/RemoteData.java

```diff
@@ -82,15 +82,16 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    // try-with-resources not used as we wish to close the remote virtual file system
```

Comment:
Is this comment correct?  Sounds like you DON'T want to close the FS.

Reply:
Good catch let me read that over; The VFS API has lots of warnings: _you must close this FS - but if you close the shared FS managed by the singleton everything will be bad forever ..._

It is a terrible API to manage with PMD

Reply:
Okay adjusted the comment to be more clear, and near the code rather than the annotation